azure-pipelines.yml

```diff
@@ -205,6 +209,24 @@ jobs:
           artifactName: $(artifact-windows-installer)
           targetPath: $(Build.StagingDirectory)/release
         condition: not(eq(variables['skip-github'], 'TRUE'))
+      - bash: |
+          set -euo pipefail
+          KEY_FILE=$(mktemp)
+          GPG_DIR=$(mktemp -d)
+          cleanup() {
+              rm -rf $KEY_FILE $GPG_DIR
+          }
+          trap cleanup EXIT
+          echo "$GPG_KEY" > $KEY_FILE
+          gpg --homedir $GPG_DIR --no-tty --quiet --import $KEY_FILE
+          cd $(Build.StagingDirectory)/release
+          # Note: relies on our release artifacts not having spaces in their
+          # names
+          for f in *; do
```

Comment:
Just to be sure I understand this correctly (might be worth documenting), this will create separate signature files that will then be automatically picked up by the release process and uploaded to github releases as well?